integr8ly/integreatly-operator
# MGDAPI-5263 - grafana: bump init plugin image

# Issue link

https://issues.redhat.com/browse/MGDAPI-5263

# What

* Use the 0.1.0 grafana init plugin image instead of using the defaulted 0.0.6 version

# Verification steps



* Install RHOAM from this branch
```
make cluster/prepare/local
make code/run
```
* Verify installation is successful
* Verify the 0.1.0 version of the grafana init plugins images is used 
```
oc get deployment grafana-deployment -n local-rhoam-customer-monitoring-operator -o json | jq '.spec.template.spec.initContainers[0].image'
```
